server/src/main/java/org/elasticsearch/action/admin/indices/alias/IndicesAliasesRequest.java

```diff
@@ -271,6 +279,9 @@ public void writeTo(StreamOutput out) throws IOException {
                 out.writeOptionalBoolean(isHidden);
             }
             out.writeStringArray(originalAliases);
+            if (out.getVersion().onOrAfter(Version.V_8_0_0)) {
+                out.writeOptionalBoolean(mustExist);
```

Comment:
The PR labels include `7.9.0` but this seems to exclude < v8.0.0. Kibana would benefit a lot from this if it was available in 7.x could this be backported?

Reply:
These conditions are typically adjusted after backporting the PR (as it otherwise breaks CI when merging into master and running BWC tests). No need to worry.